tests/image_creation.sh

```diff
@@ -346,6 +390,49 @@ test_distro_alpine()
 	run_test "${name}" "" "alpine" "no" "init"
 }
 
+# Displays a list of all distro test functions
+get_distro_test_names()
+{
+	typeset -F | awk '{print $3}' |\
+		grep "^${test_func_prefix}" | sort
+}
+
+# Displays a list of distros which can be tested
+list_distros()
+{
+	local funcs
```

Comment:
is `funcs` used here? feels like it might be a leftover.

Reply:
Good catch - removed.